compute/functions/how-to/add-a-custom-domain-name-to-a-function.mdx

```diff
@@ -0,0 +1,34 @@
+---
+meta:
+  title: How to add a custom domain to a function
+  description: This page explains how toadd a custom domain to a function
+content:
+  h1: How toadd a custom domain to a function
+  paragraph: This page explains how to add a custom domain to a function
+dates:
+  validation: 2022-02-21
+  posted: 2022-02-21
+---
+
+Each function comes with an automatically generated domain on which it is available. However it is possible to use your own domain name to address your functions to facilitate their use.
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- A running [Serverless Function](https://www.scaleway.com/en/docs/compute/functions/how-to/deploy-a-function/)
+- A domain name 
+
+</Message>
+
+1. Add a CNAME pointing to your function default URL from the [domains dashboard](/network/dns-cloud/how-to/manage-dns-records/).
```

Comment:
```suggestion
1. Add a CNAME record pointing to your function's default URL from the [domains dashboard](/network/dns-cloud/how-to/manage-dns-records/).
```

Reply:
Does the domain DNS have to be managed by SCW?

Reply:
I dont't think it has forcefully be managed by scw DNS - we can generalize this if you prefer